docs/docsite/rst/dev_guide/developing_module_utilities.rst

```diff
@@ -4,84 +4,112 @@
 Appendix: Module Utilities
 **************************
 
-Ansible provides a number of module utilities that provide helper functions that you can use when developing your own modules. The ``basic.py`` module utility provides the main entry point for accessing the Ansible library, and all Python Ansible modules must, at minimum, import ``AnsibleModule``::
+Ansible provides a number of module utilities, or snippets of shared code, that
+provide helper functions you can use when developing your own modules. The
+``basic.py`` module utility provides the main entry point for accessing the
+Ansible library, and all Python Ansible modules must, at minimum, import
+``AnsibleModule``::
 
   from ansible.module_utils.basic import AnsibleModule
```

Comment:
The minimum is actually to import anything from ```ansible.module_utils```.  You aren't required to import anything specifically from there.

So, for instance, amazon modules have started to use:
``` python
 from ansible.module_utils.aws.core import AnsibleAWSModule
```
instead of importing AnsibleModule directly.

Reply:
This change looks good.